frontend/packages/ceph-storage-plugin/src/utils/block-pool.tsx

```diff
@@ -60,18 +60,20 @@ export const PROGRESS_STATUS = (t: TFunction): ProgressStatusProps[] => [
   },
   {
     name: POOL_PROGRESS.NOTREADY,
-    icon: LockIcon,
-    desc: t(
-      'ceph-storage-plugin~The creation of an OCS storage cluster is still in progress or has failed. Please try again after the storage cluster is ready to use.',
-    ),
-    className: '',
+    icon: ExclamationCircleIcon,
+    desc: t('ceph-storage-plugin~Pool {name} got created with non ready state'),
```

Comment:
Pool CR creation is created successfully but status is a failure. If we display pool creation has failed, Then if the customer uses the same name again to create pool then the flow will get an error like pool {name} already exists. This is not correct right.

the pool is created but the status is a failure, which means it is still not ready. once PG is available status changed back to ready. So we can't display it failed

Reply:
I think its okay , one does not know it is always PGs limit. It could be anything which may lead to pool to be in "Failed" state.
```suggestion
    desc: t('ceph-storage-plugin~Pool {name} created in Failed state'),
```
Also "Ready" is a specific pool state. We should text it as "Ready". You can check with doc team for adjustments..
```suggestion
    desc: t('ceph-storage-plugin~Pool {name} got created with non Ready state'),
```

Reply:
@GowthamShanmugam @yuvalgalanti shalle we link the pool resource name `{name}` to pool's page ? 
So in case of failure, a user can check.

Reply:
It will reduce the effort of visiting to block the pool page. But we are specifying pool is created, so the user will definitely visit the pool page. I don't find any strong use case here.

Reply:
Doc team suggeston: Pool {name} created with errors

Reply:
> It will reduce the effort of visiting to block the pool page. But we are specifying pool is created, so the user will definitely visit the pool page. I don't find any strong need here.

exactly, this is to support accessibility and good to have. But I am not very keen to block the Pr for this.